### ISSUE TITLE
Improve Terrain Vehicle surface combat

### ISSUE THREAD

**Issue by Blakeley00** [Aeneas137/starflight-tlc#23]
Taking on hostile creatures with the Terrain Vehicle in TLC is very difficult and rather cumbersome. If you watch a lot of the TLC lets play videos on youtube you'll find that most first time players will lose their TV on mission 3 or so (the Zoological expedition one) due to running into a hostile creature while trying to find the one they're meant to collect. It's unlikely they'll have the funds to buy another one thus rendering their game stuck probably leading to rage quits. I know several people who have modded TLC's config files to give the TV massive armour bonuses to try and compensate for this issue.

Hostile creatures will charge at you the moment they enter your field of vision at a speed possibly 3-4x as fast as they were on Genesis and can kill the TV in only a couple of seconds. However unlike Genesis you have no dedicated fire keyboard key. You're only chance is to hold down the reverse key driving away as fast as you can while desperately trying to use the mouse interface to click on the fast moving creature, then click the scan button and then stun button (all while trying to face the gun in the right direction lol).

I discuss the above issues while playing this mission (at 29:49 mark) in my own TLC tour video here:
https://youtu.be/VL4T2DR2Xm0?t=1789

Space combat in TLC has dedicated fire keys for lasers and missiles so using one of those keys as a fire button for the TV would add a nice user friendly consistency between both combat modes and making it more fun.

A lot of players thoroughly enjoyed creature hunting & selling on Genesis due it's more 'arcade like' controls and similar controls to space combat. Whereas I don't know anyone who enjoyed creature hunting on PC SF1 due to its cumbersome controls and creature limits.

So to summarize my suggestions on this issue are:
- Add a dedicate fire keyboard key for the TV (similar to space combat). If that's too powerful consider lowering fire rate.
- Increase armour on TV or considering adding a TV armour upgrade tree like on Genesis.
- Slightly decrease movement speed of hostile creatures. At least on nearby planets to game start location.

**Comment by Anarchian** [comment]
I added Space key for firing the stunner in a revision that dead ended and it did make planet excursions more fun. That can easily be done. We can also slow down the fastest animals. But, the stunner should be slowed to about one shot per second and take into account the crew's abilities.

**Comment by raestuart** [comment]
i agree with blakeley the atv combat experience has me ignoring and avoiding biologicals
